src/SonarScanner.MSBuild.Tasks/SonarScanner.MSBuild.Tasks.csproj

```diff
@@ -36,4 +36,9 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Why only for netstandard 2.0? I think we can update the reference for all frameworks.

Reply:
With the latest update the package is working for all framework versions.